ambry-router/src/main/java/com/github/ambry/router/GetBlobOperation.java

```diff
@@ -99,6 +100,7 @@ class GetBlobOperation extends GetOperation {
   private final Map<Integer, GetChunk> correlationIdToGetChunk = new HashMap<>();
   // Callback to charge against quota for each chunk that's fetched.
   private final QuotaChargeCallback quotaChargeCallback;
+  private final GetManager getManager;
```

Comment:
I suggest we create a class, which is just a thin wrap of the Guava Cache, and we create an object of this class and pass this object to GetBlobOperation, so that we don't have to implements the same method in all these classes.

Reply:
done